src/filters/network_matchers.rs

```diff
@@ -0,0 +1,463 @@
+// ---------------------------------------------------------------------------
+// Filter matching
```

Comment:
nit: remove it or make an inner documentation comment: `//! <module description>`